pkg/remote/remote.go

```diff
@@ -126,6 +141,29 @@ func Referrers(ctx context.Context, d name.Digest, option types.RegistryOptions)
 	return nil, errs
 }
 
+func registryMirrors(hostRef name.Reference, option types.RegistryOptions) []name.Reference {
+	var mirrors []name.Reference
+
+	ctx := hostRef.Context()
+	reg := ctx.RegistryStr()
```

Comment:
nit: I thought `ctx` would be used later in the function, but actually just used to get a registry string. IMO, it's better not to define a new variable here to reduce confusion. But it's no big deal. You can decide it. 

```suggestion
	reg := hostRef.Context().RegistryStr()
```